department-of-veterans-affairs/va.gov-cms
# VACMS-5725: Add help page to menu based alias pattern.

## Description
See #5725

## Testing done
Visual / behat

## QA steps
 - As an administrator, go to `admin/config/search/path/patterns`
 - Visually verify that `documentation_page` has been added to `Menu item based` pattern.

### Definition of Done


### Select Team for PR review


### Is this PR blocked by another PR?


### Does this PR need review from a Product Owner


### CMS user-facing annoucement

Is an announcement needed to let editors know of this change?
